platform/config/amp-dev-redirects.yaml

```diff
@@ -21,3 +21,5 @@
 ### Other redirects
 # Put here redirects for pages that have been moved, etc
 # Add new entries in alphabetical order
+/examples/components/amp-autocomplete: /documentation/examples/interactivity-dynamic-content/simple_autosuggest/
+/examples/components/amp-autocomplete: /documentation/examples/interactivity-dynamic-content/autosuggest_form/
```

Comment:
redirects work the other way around:

```
 /documentation/examples/interactivity-dynamic-content/simple_autosuggest/ -> /examples/components/amp-autocomplete
```

Can you do me a favor and update the documentation to mention this. Thanks!

Reply:
Should be all set now!